packages/frontend/component/src/components/affine-banner/local-demo-tips.tsx

```diff
@@ -34,7 +30,10 @@ export const LocalDemoTips = ({
 
   return (
     <div className={styles.tipsContainer} data-testid="local-demo-tips">
-      <div className={styles.tipsMessage}>{content}</div>
+      <div className={styles.tipsMessage}>
```

Comment:
When do you plan to add to i18n?

Reply:
I’m not sure if this copy is the final version, I wasn’t too keen on adding i18n before this, but of course, we can add it now.